great-idea/index.html

```diff
@@ -12,12 +12,98 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
-</head>
+    
+    
+    <!-- Copy and paste your HTML from the first UI project here -->
+      
+  </head>
 
   <body>
+    <header>
+<nav>
+ <div class="topLine"><a href="services.html">Services</a></div>
+ <div class="topLine"><a href="services.html"> Product</a></div>
+ <div class="topLine"><a href="services.html"> Vision</a></div>
+ <div class="topLine"><a href="services.html">Features</a></div>
+ <div class="topLine"><a href="services.html">About</a></div>
+ <div class="topLine"><a href="services.html">Contact</a></div>
+</nav>
+  <div class="topLineImg"> <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> </div>
```

Comment:
make sure to style your img and set the height to auto to keep from distorting the img!